addons/knowledge/static/src/js/knowledge_renderers.js

```diff
@@ -208,6 +251,27 @@ const KnowledgeArticleFormRenderer = FormRenderer.extend(KnowledgeTreePanelMixin
         this.$('.breadcrumb').prepend(items);
     },
 
+    /**
+     * Render and mount the chatter of the current record.
+     *
+     * @private
+     * @returns {Promise}
+     */
+    async _renderChatter() {
+        if (!this.state.res_id) {
```

Comment:
Side question: is it possible ? in a new state maybe (no res_id) ? Chatter has a void chatter-like stuff when creating new records, do we support it (aka "tde has la flemme d'installer knowledge pour vérifier") ?

Reply:
yes it is possible, if the Knowledge view does not have any article, there is no state.res_id, I already protected this case in _onOpenChatter, and for now _renderChatter is not called anywhere else, but I thought to be better safe than sorry and protect it here too

Reply:
Ok, thanks for the info :)